src/rez/rezconfig.py

```diff
@@ -533,6 +533,10 @@
     "*PATH"
 ]
 
+shell_pathed_env_vars = {
```

Comment:
A bit of an explanation for the new setting is probably warranted here, no?

Reply:
Done.